external-crates/move/crates/move-compiler/src/parser/syntax.rs

```diff
@@ -3236,6 +3620,24 @@ fn parse_module(
     Ok((def, next_mod_attributes))
 }
 
+/// Skips tokens until next definition to be parsed (or EOF) is encountered. Returns `true` if no
+/// more parsing should be done (i.e., is possible) after this.
+fn next_def_fast_forward(context: &mut Context, prev_token_loc: Loc) -> bool {
```

Comment:
Can we do this with the stop set instead?

Reply:
Done!